lib/backend/report.go

```diff
@@ -230,6 +323,15 @@ func (s *Reporter) KeepAlive(ctx context.Context, lease Lease, expires time.Time
 
 // NewWatcher returns a new event watcher
 func (s *Reporter) NewWatcher(ctx context.Context, watch Watch) (Watcher, error) {
+	ctx, span := s.Tracer.Start(
+		ctx,
+		"backend/NewWatcher",
+		oteltrace.WithAttributes(
+			attribute.String("name", watch.Name),
+		),
+	)
+	defer span.End()
```

Comment:
I'm not sure how tracing fits in with watchers, as they are long running background tasks, right?